pkg/protoconv/time.go

```diff
@@ -23,6 +23,11 @@ func ConvertGoGoProtoTimeToGolangProtoTime(gogo *gogoTimestamp.Timestamp) *golan
 	}
 }
 
+// ConvertTimestampToTimeOrError converts a proto timestamp to a golang Time, or returns an error if there is one.
+func ConvertTimestampToTimeOrError(gogo *gogoTimestamp.Timestamp) (time.Time, error) {
```

Comment:
gotta say I'm not a huge fan of these Java-like names

Reply:
When we finish with proto migration to V2, we will have a cleanup phase to remove all introduced conversion functions. That will be a chance to rename (or remove) such functions.